plotting.py

```diff
@@ -174,8 +174,8 @@ def save_roc_curves(yhat, data, le, model_name):
 			)
 			)
 		pkl_dict.update(curves_dictionary)
-		print 'Plotting'+ " " + le.inverse_transform(k)
-		fig=ROC_plotter(curves_dictionary, model_name, title=le.inverse_transform(k), min_eff = 0.1, max_eff=1.0, min_rej=0, max_rej=100, logscale=True)
+		print 'Plotting'+ " " + data['LabelEncoder'].inverse_transform(k)
+		fig=ROC_plotter(curves_dictionary, model_name, title=data['LabelEncoder'].inverse_transform(k), min_eff = 0.1, max_eff=1.0, min_rej=0, max_rej=100, logscale=True)
 		fig.savefig('/Users/gigifstark/CERN_Work/HH2YYBB/roc'+ str(k)+model_name+'.pdf')

```

Comment:
don't hardcode your computer's specific path into the code, otherwise it will never run for others. Keep relative paths